libs/blocks/marquee/marquee.css

```diff
@@ -8,7 +8,7 @@
   min-height: 560px;
 }
 
-.marquee.small, .marquee.quiet, .marquee.inline, .marquee.large.compact {
+.marquee.small, .marquee.inline, .marquee.large.compact {
```

Comment:
What is this update about? Doesn't seem to relate to the dark/light theming.

Reply:
This is related to the min-height adjustments requested in [the linked Jira ticket](https://jira.corp.adobe.com/browse/MWPW-137523) in the description

Reply:
Got it. Thanks! That ticket is a mess... Makes quality control very difficult.

Reply:
With that said, that makes the default `quiet` variant taller (560px) on any page it's used. I assume to get back to the smaller height (360px) the `small` variation will need to be added, correct?

You can see this in play on the bacom sample page provided.